sdk/textanalytics/Azure.AI.TextAnalytics/tests/SessionRecords/RecognizeHealthcareEntitiesTests/RecognizeHealthcareEntitiesBatchWithCancellationAsyncAsync.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
interesting... do you know why the test doesn't have a sync equivalent version of the recording?